Cargo.toml

```diff
@@ -73,6 +73,7 @@ phf = { version = "0.11.2", features = ["macros"] }
 sha2 = { version = "0.9", optional = true }
 serde = { version = "1.0", features = ["derive"], optional = true }
 serde_with = { version = "3.7.0", optional = true }
+visibility = "0.1.1"
```

Comment:
🪧 This is a very light dependency that makes it possible to use `#[cfg_attr(...)]` to modify the visibility of a type or `impl` block based on the enabled features.